plugins/hydra_ax/README.md

```diff
@@ -1,7 +1,19 @@
-# Hydra example Launcher plugin
+# Hydra Ax Launcher plugin
 
-This plugin provides an example for how to write a custom Sweeper for Hydra.
+This plugin provides a mechanism for Hydra applications to use the [Adaptive Experimentation Platform aka Ax](https://ax.dev/). Ax can be used to optimize any kind of experiment - machine learning experiments, A/B tests and simulations. 
+
+The plugin can be installed by running the following command in the `plugins/hydra_ax` directory:
+
+```
+python setup.py install
+```
+
+We include an example of how to use this plugin. The file `plugins/hydra_ax/example/banana.py` implements the [Rosenbrock function (aka Banana function)](https://en.wikipedia.org/wiki/Rosenbrock_function). Ax Platform expects two return values from the function (that is being evaluated/optimized) - mean and SEM value. For more details on this, refer the [Ax documentation](https://ax.dev/tutorials/gpei_hartmann_service.html#3.-Define-how-to-evaluate-trials). Since we are using a deterministic function, we invoke the function only once, return the function value as the mean and the SEM value as 0.
```

Comment:
I think we should clean this up and return a single value from the function (handling the extra crap in the plugin).

Reply:
We do not even have to handle the SEM value. The [Ax documentation](https://ax.dev/docs/trial-evaluation.html#evaluation-function) mentions that the evaluate function `can also return only the mean as a float, in which case Ax will treat SEM as unknown and use a model that can infer it.` So we can just return the mean value. I will update this.

Reply:
Fixed in : https://github.com/facebookresearch/hydra/pull/312/commits/08675cb8306a999b7939e11f6417bad01e17f0f9